awx/main/dispatch/periodic.py

```diff
@@ -35,7 +36,11 @@ def run():
                         # If the database connection has a hiccup, re-establish a new
                         # connection
                         conn.close_if_unusable_or_obsolete()
-                    self.run_pending()
+                    GuidMiddleware.set_guid(GuidMiddleware._generate_guid())
```

Comment:
Every time the periodic background scheduler wakes up, set a new GUID so it's available in the current thread when .apply_async() is called to schedule work.  In this way, we can map beat wake's w/ tasks that get scheduled (so you can see that a task was `.apply_async()` via our periodic task *or* that is originated in some HTTP request.